### DISCUSSION TITLE
三分鐘熱度是壞事嗎?

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1364] [General]
三分鐘熱度常被認為是一個缺點，但真的是這樣嗎?
許多人會覺得三分鐘熱度的自己做甚麼事都堅持不久，常常半途而廢、設定的目標都達不成，然後開始懊惱怎麼都做不到。其實三分鐘熱度這個特質能創造更多可能性。
利用好奇心嘗試感興趣的事:擁有這特質的人願意嘗試不同領域或喜歡追求潮流，吸收不同方面的知識來充實自己。
找到真正想長久發展的:嘗試過許多感興趣的事後，可能就會找到真正想繼續做下去的。就算沒有找到也沒有關係，整合不同領域的資訊，並結合兩種以上不同興趣，也許會有不一樣的火花產生!
延續對事情的熱情，將大目標拆成小目標，並將過程記錄下來。回頭看看做了很多努力的自己，會發現自己累積了很多東西，也累積了成就感!!

**Comment by harrycoconut** [comment]
我本身也是個三分鐘熱度的人，但我並不認為這是壞事，反而是一件能讓我更廣泛接觸新鮮事物的習慣。有些三分鐘熱度的事物到最後可能變成三小時熱度，甚至變成我每天都會做事，like彈吉他，一開始我也是抱著嘗試的心情，但到了最後彈吉他變成了我的daily routine😌

**Comment by tower20021202** [comment]
我也時常做事只有三分鐘熱度，但這至少代表能夠多接觸到新的事物，搞不好就因為那三分鐘而愛上這件事，畢竟任何事都是開始最難。

**Comment by shuxian12** [comment]
我自己親生體驗就是，小時候常常會三分鐘熱度想要去學某種東西，但當學到一定的程度，沒辦法輕易學會之後就會想要放棄，或是可能因為學樂器會需要檢定，勢必就需要一直練習，這時候就會覺得很煩，然後就不會回家練習，都在教室的時候才練。
但長大之後卻會對那些小時候放棄的事物又感興趣，或許是因為沒有人盯著你或要求你去學習了吧！！

**Comment by juliaouo** [comment]
我認為三分鐘熱度的人能堅持做一件事代表他對那件事一定非常的喜愛，反而更能發掘自己的興趣及能力。但是如果是一件需要負責的嚴肅事情，三分鐘熱度不是一件好事，可能會導致事情做不完，做不好，所以有好有壞吧！

**Comment by Tzuhsuannnnn** [comment]
我也是一個三分鐘熱度的人，我認為他是個壞習慣，因為很多事情要產生很大的熱忱，或學到深入的東西，需要投入過多一點的時間，而非簡單幾天就可以達到的。

**Comment by Miinteaa** [comment]
三分鐘熱度並非完全是一件壞事，如同原貼文所說，這能讓自己嘗試各種未接觸過的領域，若在這個過程中人們獲得了成就感，那麼就有更大的機會持續進行下去，成為一項新的技能；若沒有獲得成就感，那麼可能也代表自己不適合這項事情，或者做的不快樂，能夠更了解自己的喜好及適合自己的生活方式。

**Comment by ALAN30359704** [comment]
我覺得人的耐心與精力是有限的，三分鐘熱度的原因可能是來自於我們生活中有太多的壓力與資訊，我們容易會對突然熱衷的事情感到興趣，無法持續保持熱情或興趣，但這也不是我們可以怠惰的藉口，反而我們要思考的是甚麼是我們真正想要的。

**Comment by KozakHou** [comment]
儒貝爾曾說過一句意義深遠的話，冷酷無情的理智是一把搗毀之外毫用處的錘子。它有時就像冷酷的心一樣有害和可恨。但願各位能從這段話中獲得心靈上的滋長。我想，把三分鐘熱度的意義想清楚，對各位來說並不是一件壞事。那麼，這種事實對本人來說意義重大，相信對這個世界也是有一定意義的。

三分鐘熱度的出現，必將帶領人類走向更高的巔峰。三分鐘熱度的出現，重寫了人生的意義。我們都知道，只要有意義，那麼就必須慎重考慮。伊朗在過去曾經講過，聰明才智是撥動社會的槓桿。這句話改變了我的人生。

毛澤東說過一句很有意思的話，以絕對的總的宇宙發展過程中，各個具體過程的發展都是相對的，因而在絕對真理的長河中，人們對於在各個一定發展階段上的具體過程的認識只具有相對的真理性。這句話看似簡單，卻埋藏了深遠的意義。

**Comment by a3240281370** [comment]
從某種程度上說，三分鐘熱度可能被視為一種負面的特徵，因為它意味著缺乏長期的承諾和穩定性。這可能對個人的目標達成、人際關係以及其他方面產生負面的影響。
然而，每個人都有不同的興趣和熱情，而且興趣的持久性也會因人而異。有些人天生喜歡追求新鮮感，他們可能更容易在不同的事物之間切換，這並不一定是壞事。但如果一個人在各個方面都只有短暫的興趣，而且從不持久地追求一個目標或興趣，這可能會對他們的生活和發展產生一些不利的影響。
因此，我覺得判斷是否三分鐘熱度是壞事，取決於具體情況和個人的價值觀。重要的是要找到一種平衡，並確保在追求興趣和目標的同時，也保持著持久的承諾和努力。

**Comment by Jamesmoneya** [comment]
三分鐘熱度就是可以讓你嘗試任何擬向要得東西，當你找到你真的喜歡的東西的時候，你就不會是三分鐘熱度了，所以趕快去多多嘗試吧

**Comment by aa103967622** [comment]
我認為三分鐘熱度是嘗試新事物必經的過程，其實就是在多方嘗試中，意識到這件事情並不適合自己，或是興趣不大，我們選擇了停止繼續做這件評斷為沒有價值的事情，不讓自己浪費生命，或許在別人眼中看起來像半途而廢，但其實我們只是在尋找熱忱的路上而已。

**Comment by hardtagottaname** [comment]
我認為三分鐘熱度至少會比不嘗試了解新事物好。三分鐘熱度通常是因為我們一開始對一件事情感到好奇，試做了一下之後才發現並不如預想的那般。我也常常會對一些事只保持三分鐘熱度，但也有一些事物是我原本不抱有希望，可在嘗試過後，才發現事情是我所希望和想要的，從而慢慢培養成興趣。因此，我覺得三分鐘熱度對我而言不是貶義的代名詞，反而是願意嘗試及理解，屬於正向的詞。

**Comment by Superlan8** [comment]
我覺得三分鐘熱度對於開發興趣來說是好的，就像作者說的，可以嘗試各種不同的東西，體驗不同的事物，但針對學習來說，三分鐘熱度確實是一件蠻糟的事，沒有學習的耐心，往往虎頭蛇尾，這是典型三分鐘熱度的長相。綜合以上，我還是認為這是貶義詞，畢竟能在三分鐘熱度找到真正興趣的人遠小於半途而廢的人。

**Comment by Alexgns** [comment]
三分鐘熱度通不一定是壞事，因為人們對不同事物的興趣可能會有變化，然而，若三分鐘熱度成為一種長期的模式，可能會對個人的成長和成功產生負面影響。持續不斷地轉換興趣和活動，可能意味著缺乏專注和耐心，無法深入探索和精通任何一項技能或領域，因此，重要的是找到平衡點。保持開放心態，探索不同的興趣和活動是有益的，但同時也要培養專注和持續學習的能力。了解自己的興趣和目標，並為之付出持久的努力，有助於個人的成長和達成長期目標。

**Comment by pigeon9975** [comment]
我認為三分鐘熱度不是壞事，這樣可以在短時間內透過多方嘗試，了解自己真正喜歡的東西，比起半途而廢的說法，我覺得尋找自我似乎更適合這一類人，畢竟當自己遇到自己想做的事情時，就會持之以恆，付出時間與精力想把他做得更好，這時候還有誰會說自己是三分鐘熱度呢？

**Comment by pudding0803** [comment]
我覺得人生本來就有很多不同的事物可以嘗試，同一個事情或興趣並不是每個人都適合的，我認為如果對一件事情出現了三分鐘熱度的情況，或許就代表自己可能對這件事並不是很有興趣，也不需要浪費太多時間，或是在沒興趣的事物上執著太久，可以再去嘗試看看新的事物😇！

在這種篩選自己喜歡或不喜歡的事物的過程中，我們也在漸漸了解自己！

**Comment by 1104030360** [comment]
對我來說，三分鐘熱度不一定是壞事，我會因為三分鐘熱度嘗試到很多不同的事，我認為不一定要做很久，可以制定一個較快達成的小目標，這樣即使是三分鐘熱度也是心安理得。

**Comment by xiang305** [comment]
我認為三分鐘熱度並非是壞事，俗話說：「萬事起頭難」，但這樣的人卻能保有熱情地去嘗試各種不同的事物，雖然最後每件事不一定都會堅持下去，不過一旦發現自己真正熱愛的事物，便能因此一展長才

**Comment by quan0715** [comment]
我認為至少我們願意去嘗試那三分鐘，畢竟找尋興趣就如同大海老針一般，如果真的堅持下去了，那他就是你賺到，如果沒有，那也就頂多算損失一些金錢與時間，所以結論是三分鐘熱度是必然的結果。

**Comment by Aaron0810** [comment]
我個人是覺得，反過來說，三分鐘熱度代表你是一個很有好奇心的人，而這件事絕對不是不好的，有好奇心代表你願意嘗試更多的事情，而且也有人說過三分鐘熱度只是因為你還沒找到真正喜歡的事而已，或許在找到真正喜歡的事情之後，就會蛻變了，因此我覺得三分鐘熱度只是在找到興趣的其中一個過程。

**Comment by Elaine2899** [comment]
我覺得三分鐘熱度的人有個優點是你會願意嘗試很多東西，因此你會比別人更知道什麼適合自己、自己喜歡什麼，因此這表面上是個缺點，但也有隱藏的優點。
